mp3lyrics.rb

```diff
@@ -8,18 +8,46 @@ def to_b(string)
   !(string =~ /^(true|t|yes|y|1)$/i).nil?
 end
 
+def usage_message(overide_options, wiki_options)
+  'Usage: ./mp3lyrics.rb <dir> [-overide ' + overide_options.join("/") + '] [-wiki ' + wiki_options.join("/") + ']'
+end
+
 override = false
+wiki_to_use = nil
 dir = nil
 
-case ARGV.length
-when 1
-  dir = ARGV[0]
-when 2
-  override = to_b(ARGV[1])
-  dir = ARGV[0]
-else
-  puts 'Usage: ./mp3lyrics.rb <dir> [override]'
-  exit
+dir = ARGV[0]
```

Comment:
you broke the check that there is at least one argument is present - otherwise the usage message is printed